### PR TITLE
Code refactory and UI improvements for TwoPage sample

### PR DESCRIPTION
**Pull request by ancirja-m**

### CODE REVIEW

**Review by bimiron — APPROVED** [review]
Nice work!

File: TwoPage/src/androidTest/java/com/microsoft/device/display/samples/twopage/TwoPageModeOrientationTest.kt [hunk 1]
```diff
@@ -31,7 +31,7 @@ class TwoPageModeOrientationTest {
     private val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
 
     @get:Rule
-    val activityRule = ActivityTestRule<MainActivity>(MainActivity::class.java)
+    val activityRule = ActivityTestRule<TwoPageActivity>(TwoPageActivity::class.java)
```
Comment (CesarValiente) [inline TwoPage/src/androidTest/java/com/microsoft/device/display/samples/twopage/TwoPageModeOrientationTest.kt:34]: Are you going to do the same regarding replacing `ActivityTestRule` like previous PRs? I think we should ;-)

File: TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/PagerAdapter.kt [hunk 2]
```diff
@@ -3,20 +3,31 @@
...
 import android.util.SparseArray
+import androidx.core.util.forEach
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentManager
 import androidx.fragment.app.FragmentPagerAdapter
+import com.microsoft.device.display.samples.twopage.fragments.ScrollingContent
 
 internal class PagerAdapter(
-    fm: FragmentManager?,
+    fragmentManager: FragmentManager,
```
Comment (CesarValiente) [inline TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/PagerAdapter.kt:17]: love it! these small things, that somehow you understand but that maybe requires a bit of time to guess what it was are good candidates to be changed. `fragmentManager` is a very better name, more descriptive and clear.

File: TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/extensions/ScreenInfo.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+import android.view.Surface
+import com.microsoft.device.dualscreen.ScreenInfo
+
+/**
+ * Returns [true] if the screen is in portrait mode, [false] otherwise
+ */
+val ScreenInfo.isInPortrait: Boolean
+    get() = getScreenRotation() == Surface.ROTATION_90 ||
+        getScreenRotation() == Surface.ROTATION_270
```
Comment (CesarValiente) [inline TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/extensions/ScreenInfo.kt:17]: Github highlights that there is not a new line after this, probably good to add it. Even though we have that disabled with ktlint, here looks better with a new line.

File: TwoPage/src/androidTest/java/com/microsoft/device/display/samples/twopage/TwoPageModeOrientationTest.kt (hunk 1 above)
Comment (ancirja-m) [reply TwoPage/src/androidTest/java/com/microsoft/device/display/samples/twopage/TwoPageModeOrientationTest.kt]: I already did this in another commit, please check.

File: TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/extensions/ScreenInfo.kt (hunk 3 above)
Comment (ancirja-m) [reply TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/extensions/ScreenInfo.kt]: Done

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Andrei!

**Pull request merged**